scripts/validate-requirements.sh

```diff
@@ -0,0 +1,221 @@
+#!/bin/bash
+
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Usage:
+# bash scripts/validate-requirements.sh "END_USER_EMAIL" "ORGANIZATION_ID" "BILLING_ACCOUNT_ID"
+
+# -------------------------- Variables --------------------------
+# Expected versions of the installers
+TF_VERSION="0.13.7"
+GCLOUD_SDK_VERSION="319.0.0"
+GIT_VERSION="2.25.1"
+# User Inputs
+END_USER_CREDENTIAL="$1"
+ORGANIZATION_ID="$2"
+BILLING_ACCOUNT="$3"
+
+ERRORS=""
+# -------------------------- Funcions ---------------------------
+
+# Compare the two float numbers
+function compare_version(){
+    # echo "comparing $1 and $2"
+    if [[ "$1" == "$2" ]]; then
+        return 0
+    fi
+
+    local IFS=.
+    local i
+    local version1=("$1")
+    local version2=("$2")
+    for ((i=${#version1[@]}; i<${#version2[@]}; i++))
+    do
+        version1[i]=0
+    done
+    for ((i=0; i<${#version1[@]}; i++))
+    do
+        if [[ -z ${version2[i]} ]]
+        then
+            version2[i]=0
+        fi
+        if [[ ${version1[i]} > ${version2[i]} ]]
+        then
+            return 1
+        fi
+        if [[ ${version1[i]} < ${version2[i]} ]]
+        then
+            return 2
+        fi
+    done
+    return 0
+}
+
+# Validate the Terraform installation and version
+function validate_terraform(){
+
+    if [ ! "$(command -v terraform )" ]; then
+        echo "Terraform not found."
+        echo "Visit https://learn.hashicorp.com/tutorials/terraform/install-cli and follow the instructions to install Terraform."
+        ERRORS+=$'Terraform not found\n'
+    else
+        TERRAFORM_CURRENT_VERSION=$(terraform version -json | jq -r .terraform_version)
+        compare_version "$TERRAFORM_CURRENT_VERSION" "$TF_VERSION"
```

Comment:
Terraform version comparation is different from the others:
- `terraform` version must be `==` 
- `gcloud` and `git` must be `>=`